charts/temporal/values.yaml

```diff
@@ -242,6 +242,35 @@ server:
     containerSecurityContext: {}
     topologySpreadConstraints: []
     podDisruptionBudget: {}
+  internal-frontend:
+    # Enable this to create internal-frontend
+    enabled: false
+    service:
+      # Evaluated as template
+      annotations: {}
+      type: ClusterIP
+      port: 7233
+      membershipPort: 6933
+      httpPort: 7243
```

Comment:
Should these ports be 6936 and 7236 by default to match https://github.com/temporalio/temporal/blob/main/docker/config_template.yaml#L283-L284

Reply:
Ports 6936 and 7236 also matches the suggestion in [the release notes for version 1.20](https://github.com/temporalio/temporal/releases/tag/v1.20.0) (which introduced the internal-frontend).